src/utils_request.js

```diff
@@ -119,6 +119,7 @@ export const requestAsBrowser = async (options = {}) => {
         headerGeneratorOptions: ow.optional.object,
         stream: ow.optional.boolean,
         decodeBody: ow.optional.boolean,
+        sessionToken: ow.optional.object,
```

Comment:
should be also added to the `RequestAsBrowserOptions` jsdoc to have it in the docs